src/block.js

```diff
@@ -4,20 +4,34 @@ var fastMerkleRoot = require('merkle-lib/fastRoot')
 var typeforce = require('typeforce')
 var types = require('./types')
 var varuint = require('varuint-bitcoin')
+var networks = require('./networks')
+var coins = require('./coins')
 
 var Transaction = require('./transaction')
 
-function Block () {
+function Block (network) {
+  typeforce(types.maybe(types.Network), network)
+  network = network || networks.bitcoin
```

Comment:
We've been burned by default values before. Wouldn't it be better to fail when no network is provided?

Reply:
The problem with changing this now is that the behavior will be inconsistent across the lib and may break other projects (like our). It was a design decision that by default it works for bitcoin.